### PR TITLE
serverURL is actually not required in CLI

### PR DESCRIPTION
**Pull request by flovilmart**
small nit for the CLI error

### CODE REVIEW

**Comment by gfosco** [comment]
Why don't we require it? If you have any cloud code at all which calls back in to Parse, it's necessary, so it seems like a common snag people will hit.

**Comment by flovilmart** [comment]
because it's inferred by the default port (1337) and default mount path (/parse).

File: src/cli/parse-server.js [hunk 1]
```diff
@@ -56,7 +56,7 @@ if (!options.serverURL) {
 if (!options.appId || !options.masterKey || !options.serverURL) {
```
Comment (gfosco) [inline src/cli/parse-server.js]: this needs to be updated too right?

Comment (flovilmart) [inline src/cli/parse-server.js]: we need serverURL, but we just don't need to say it

**Pull request merged**